notebooks/github_pr_creator_agent.ipynb

```diff
@@ -0,0 +1,930 @@
+{
```

Comment:
<p>On the website, this markdown output is rendered as part of the recipe. We need to either copy and put it as a text or remove it completely</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/183/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>